grc/core/FlowGraph.py

```diff
@@ -216,7 +222,7 @@ def get_elements(self):
         return elements
 
     def children(self):
-        return itertools.chain(self.iter_enabled_blocks(), self.connections)
+        return itertools.chain(self.iter_all_blocks(), self.connections)
```

Comment:
Why not pass `self.blocks` directly?

Reply:
Yes, you are that works and is much easier